porting/examples/freertos_nrf5_sdk/main.c

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
Could this example be made completely portable and put in examples/common or examples/dummy?